server/command.go

```diff
@@ -442,67 +441,25 @@ func executeUninstallCloud(p *Plugin, c *plugin.Context, header *model.CommandAr
 		return p.responsef(header, "No current Jira instance to uninstall")
 	}
 
-	jci, ok := ji.(*jiraCloudInstance)
-	if !ok {
-		return p.responsef(header, "The current Jira instance is not a cloud instance")
-	}
-
-	if jiraURL != jci.GetURL() {
-		return p.responsef(header, "You have entered an incorrect URL. The current Jira instance URL is: `"+jci.GetURL()+"`. Please enter the URL correctly to confirm the uninstall command.")
-	}
-
-	err = p.instanceStore.DeleteJiraInstance(jci.GetURL())
-	if err != nil {
-		return p.responsef(header, "Failed to delete Jira instance "+ji.GetURL())
-	}
-
-	// Notify users we have uninstalled an instance
-	p.API.PublishWebSocketEvent(
-		wSEventInstanceStatus,
-		map[string]interface{}{
-			"instance_installed": false,
-			"instance_type":      "",
-		},
-		&model.WebsocketBroadcast{},
-	)
-
-	const uninstallInstructions = `Jira instance successfully disconnected. Go to **Settings > Apps > Manage Apps** to remove the application in your Jira Cloud instance.`
-
-	return p.responsef(header, uninstallInstructions)
-}
-
-func executeUninstallServer(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
-	authorized, err := authorizedSysAdmin(p, header.UserId)
-	if err != nil {
-		return p.responsef(header, "%v", err)
-	}
-	if !authorized {
-		return p.responsef(header, "`/jira uninstall` can only be run by a System Administrator.")
-	}
-	if len(args) != 1 {
+	instanceType := args[0]
+	var ok bool
+	if instanceType == "cloud" {
```

Comment:
nit: a `switch args[0] {` would look better here, perhaps?

Reply:
It does, indeed.